docs/installation/pagerduty/index.md

```diff
@@ -0,0 +1,126 @@
+   1. Enter **Name**. We suggest to put a cluster name, for example: "Production EU".
+   2. Click **Next** to continue.
+2. Select Escalation Policy that suits you best. Click **Next** to continue.
+3. Select the **Events API V2** integration.
+
+   ![pager_duty_integration.png](./assets/pager_duty_app_integration.png)
+
+4. Click **Create Service**.
+5. Now you will be in the service’s Integrations tab, where you’ll find the **integration key**.
+6. Copy the **integration key**.
+
+Proceed with the next section.
+
+### Finalize Botkube Cloud instance configuration
+
+Go back to the Botkube Cloud instance creation.
+
+1. In step 2, specify configuration details:
+
+   - **Integration key:** The integration key generated by Pager Duty in the previous section.
+
+   ![Botkube Cloud Instance Configuration](../assets/undraw_under_construction.svg "Botkube Cloud Instance configuration")
+
+2. Add source plugins you want to enable in your Botkube instance and click `Next` button.
+
+   ![Source Plugins](../assets/undraw_under_construction.svg "Source plugins")
+
+3. Click `Apply Changes` button to update Botkube Cloud instance.
+
+   ![Cloud Instance Create](../assets/undraw_under_construction.svg "Cloud Instance create")
+
+## Using Botkube Cloud with PagerDuty Integration
+
+From now on, all events emitted by enabled sources are routed directly to created PagerDuty service using the [Events API v2](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgw-events-api-v2-overview). Botkube follows the best practices by recognizing different event types and sending them to PagerDuty accordingly.
+
+![pager_duty_app_alert_and_changes.png](assets/pager_duty_app_alert_and_changes.png)
+
+### Alerts
+
+An [alert](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgx-send-an-alert-event) indicates a problem within a monitored system. Botkube forwards the following types of alerts to PagerDuty:
+
+- [Kubernetes'](../../configuration/source/kubernetes.md#event-types) `error` and `delete` events.
+- All configured [Prometheus](../../configuration/source/prometheus.md) alerts.
+- The [ArgoCD](../../configuration/source/argocd.md) application events including synchronization failures and health status degradation.
+
+To see the alerts in the PagerDuty web app, click **Incidents** tab and select **Alerts**:
+
+![pager_duty_app_alert.png](assets/pager_duty_app_alert.png)
+
+### Changes
+
+A [change](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgy-send-a-change-event) indicates a non-problematic update within a system. Botkube forwards the following types of change events to PagerDuty:
+
+- [Kubernetes'](../../configuration/source/kubernetes.md) `create` and `update` events.
+- The [ArgoCD](../../configuration/source/argocd.md) application events including creation, updates, deployment, and deletion. Additionally, the synchronization statuses include running, unknown, and succeeded events.
```

Comment:
WDYT about something like that:
```suggestion
- - **[Kubernetes](../../configuration/source/kubernetes.md) plugin:** `create` and `update` events.
- **[ArgoCD](../../configuration/source/argocd.md) plugin:** application events including creation, updates, deployment, and deletion. Additionally, the synchronization statuses include running, unknown, and succeeded events.
```

If you like it, please also apply it in the "Alerts" section 👍